WcaOnRails/spec/controllers/committees_controller_spec.rb

```diff
@@ -0,0 +1,221 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe CommitteesController, type: :controller do
+  let(:committee) { FactoryGirl.create :committee }
+  let(:committee_with_team) { FactoryGirl.create :committee, :with_team }
+
+  describe "GET #index" do
+    context "when not signed in" do
+      sign_out
+
+      it 'shows the committee index page' do
+        get :index
+        expect(response).to render_template :index
+      end
+    end
+
+    context "when signed in as admin" do
+      sign_in { FactoryGirl.create :admin }
+
+      it 'shows the committee index page' do
+        get :index
+        expect(response).to render_template :index
+      end
+    end
+
+    context 'when signed in as a regular user' do
+      sign_in { FactoryGirl.create :user }
+
+      it 'shows the committee index page' do
+        get :index
+        expect(response).to render_template :index
+      end
+    end
```

Comment:
But I guess you wanted to make all tests uniformed, so I don't have strong feeling for removing this if you like it =)